Detections/MultipleDataSources/MalformedUserAgents.yaml

```diff
@@ -49,3 +49,4 @@ query: |
   or UserAgent matches regex  @"MSIE(?:\d|.{1,5}?\d\s;)"
   | summarize by TimeGenerated, UserId, ClientIP, UserAgent, OperationName, RecordType
```

Comment:
Probably makes sense to use min() and max() TimeGenerated here to reduce the result set.  Might even consider a makeset for the OperationName and RecordType if it makes sense.  Also, makeset(UserAgent) and then use dcount(UserAgent) to indicate multiple UserAgents used by the User which might be of interest to know.

Reply:
Done. Thanks